Automattic/wp-calypso
# Update progress bar consistency with updated tasks

#### Proposed Changes

* Changes to fix progress bar percentage and visuals. These changes will calculate the checklist percentage based on the task list after they have been fully updated/enhanced. Prior to these changes, the progress bar percentage was calculated based on the initial state of the checklist for each flow.



#### Testing Instructions



* Check out this branch and execute `yarn start` command
* Create a new `link-in-bio` site. Visit https://wordpress.com/hp-2022-tailored-flows/ and scroll down to the` Put a dent in the web.` In 3 minutes flat. subheading. You should end up on the launchpad at the end of the flow.
* Switch to local calypso and load the launchpad page again (http://calypso.localhost:3000/setup/launchpad?flow=link-in-bio&siteSlug=YOUR_SITE_SLUG)
* The `Add links` task should be enabled and the progress bar percentage should be: **`60%`**
<img width="436" alt="image" src="https://user-images.githubusercontent.com/10482592/187965241-e1ef16de-abbb-4156-a38e-277e607f0e83.png">

* Click the `Add links` task
* Navigate back to the launchpad using the navigation sidebar in the site editor
*  The `Add links` task should be disabled and the progress bar percentage should be: **`80%`**
<img width="442" alt="image" src="https://user-images.githubusercontent.com/10482592/187965071-877faf1e-df73-4f38-b524-694c4072e1c2.png">

* Navigate to the newsletter flow (http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=YOUR_SITE_SLUG)
* Ensure the progress bar percentage is **`75%`**

#### Pre-merge Checklist






Related to #
